query-node/mappings/src/membership.ts

```diff
@@ -193,7 +193,7 @@ async function createNewMemberFromParams(
     rootAccount: rootAccount.toString(),
     controllerAccount: controllerAccount.toString(),
     handle: bytesToString(memberHandle),
-    handleRaw: memberHandle.toHex(),
+    handleRaw: Buffer.from(memberHandle.toU8a(true)).toString('hex'),
```

Comment:
maybe refactor all the `handle`-setting logic into one function and use it in both places? Since you are setting it in 2 different ways

Reply:
also does it automatically sets the '0x' in front of the string?

Reply:
its doesn't, and I actually pushed a later commit to fallback to simply using `.toHex()`